presto-main/src/main/java/io/prestosql/operator/aggregation/LongSumAggregation.java

```diff
@@ -31,27 +32,34 @@
     private LongSumAggregation() {}
 
     @InputFunction
-    public static void sum(@AggregationState NullableLongState state, @SqlType(StandardTypes.BIGINT) long value)
+    public static void sum(@AggregationState LongLongState state, @SqlType(StandardTypes.BIGINT) long value)
```

Comment:
this actually causes significant perf regression as long long stats is much bigger to serialize, cc @martint @dain

Reply:
I agree, and just filed an issue to showcase the problem #18818